packages/cashier-v2/src/constants/constants.ts

```diff
@@ -0,0 +1,179 @@
+import { TMT5LandingCompanyName } from '../hooks/types';
+
+export const walletIconMapper = {
```

Comment:
walletIcons or WALLET_ICONS ?
I tend to avoid word "mapper" as it triggers ppl later down the line to use that object as container for anything,

Reply:
Oh, nice catch. This is not part of cashier-v2. Removing it 👍